itou/approvals/tests.py

```diff
@@ -913,6 +913,30 @@ class SuspensionModelTest(TestCase):
     Test Suspension model.
     """
 
+    def test_clean(self):
+        today = timezone.now().date()
+        start_at = timezone.now().date() - relativedelta(months=2)
+        end_at = start_at + relativedelta(months=2)
+        approval = ApprovalFactory(start_at=start_at, end_at=end_at)
+        suspension = SuspensionFactory(approval=approval)
+
+        # test the when suspension is out of boundaries
+        suspension.start_at = start_at - relativedelta(months=1)
```

Comment:
Ce test est fragile. Pourquoi ? Car le jour où `MAX_RETROACTIVITY_DURATION_DAYS` ne vaudra plus 30 jours mais 40, ton test sera rouge au lieu d'être vert.
Je suggère de rendre la période dynamique.

```suggestion
        suspension.start_at = start_at - relativedelta(days=Suspension.MAX_RETROACTIVITY_DURATION_DAYS + 1)
```

Reply:
t'as totalement raison 🙏